data/VT/incentives.json

```diff
@@ -218,7 +218,8 @@
       "rebate"
     ],
     "items": [
-      "ducted_heat_pump"
+      "ducted_heat_pump",
+      "ductless_heat_pump"
     ],
     "program": "vt_greenMountainPower_greenMountainPowerDuctedHeatPumpRebate",
```

Comment:
This program's name still says "Ducted Heat Pump Rebate". (It's defined in `src/data/programs/vt_programs.ts`.) Could you:

- rename the program key, here and in vt_programs.ts, to remove the word "ducted";
- remove the word "Ducted" from the user-visible name in vt_programs.ts?

The `*_programs.ts` files were originally generated from spreadsheets by script (`scripts/generate-misc-state-data.ts`), but that was basically a one-time thing when the state was first added, and now the VT spreadsheet has drifted enough that the script currently errors out. So I think hand-editing to fix this is the way forward. (Hopefully the spreadsheets' days are numbered anyway.)